rtil/src/threads/ue/rebo/rebo_init.rs

```diff
@@ -1538,3 +1542,11 @@ fn timer_get_segments() -> Vec<Segment> {
 fn timer_get_attempt_count() -> u32 {
     Run::attempt_count()
 }
+#[rebo::function("Tas::timer_save_splits")]
+fn timer_save_splits(path: String) -> Result<(), SplitsSaveError> {
+    Run::save_splits(path)
+}
+#[rebo::function("Tas::timer_load_splits")]
+fn timer_load_splits(path: String) -> Result<(), SplitsLoadError> {
+    Run::load_splits(path)
+}
```

Comment:
I just realized that we don't actually verify the path in any way! This can be used to overwrite / read any file by someone with malicious intent and should thus be avoided.

Check how the other store and load functions work. They take a filename, sanitize it, then join it with the corresponding folder.

<https://github.com/oberien/refunct-tas/blob/37224fa122bed69cc5322c01892c57b233d257f0/rtil/src/threads/ue/rebo/rebo_init.rs#L471-L477>